src/Components/Artist/CollectionsRail/CollectionEntity.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, color, Flex, Link, Sans, Serif } from "@artsy/palette"
+import currency from "currency.js"
+import React, { SFC } from "react"
+import { data as sd } from "sharify"
+import styled from "styled-components"
+
+export interface CollectionProps {
+  collection: any
+}
+
+export const ArtistCollectionEntity: SFC<CollectionProps> = ({
+  collection,
+}) => {
+  const { price_guidance, slug, title, headerImage } = collection
+  const formattedTitle = title.split(": ")[1] || title
+
+  return (
+    <Box width="100%" pr={2}>
+      <StyledLink href={`${sd.APP_URL}/collection/${slug}`}>
+        <ImgWrapper pb={1}>
+          <ImgPlaceholder headerImage={headerImage} />
```

Comment:
I know we haven't finalized the data, but I wonder if it's worth switching the structuring of the headerImage to be some sort of collection/array property on the `collection` object (i.e. `collection.railImages`). These will be the icon images or curated in some cases, correct? It may not be worth tackling in this PR but is something to keep in mind.

Reply:
If you think necessary to change will just remove the image and leave a gray box for now, not sure what the field name or shape will look like so don't want to over-engineer.

Reply:
Yeah that makes sense. I think it's fine to leave it as is for now.

Reply:
Went ahead and removed to remove any ambiguity, added a comment that background URL should be updated when fields are available.